lazy_tensor_core/lazy_tensor_core/csrc/init_python_bindings.cpp

```diff
@@ -18,6 +18,8 @@
 #include <vector>
 
 #include <torch/csrc/lazy/core/lazy_graph_executor.h>
+#include "c10/core/DeviceType.h"
+#include "lazy_tensor_core/csrc/ops/optim.h"
```

Comment:
did you forget to include this file in the PR?

Reply:
lol, yes, re-pushed.